packages/react-components/src/components/CameraButton.tsx

```diff
@@ -149,34 +161,49 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     strings.tooltipDisabledContent = strings.tooltipVideoLoadingContent;
   }
 
+  const toggleAnnouncerString = useCallback(() => {
```

Comment:
toggling may work most of the time, but in theory could be out of sync. If we don't account for all edge cases then this could be out of sync with the button state (thinking of things like our app is backgrounded and another app uses the camera thus making our app lose it, then the user returns to the app, will this become out of sync? more over -- can we catch every edge case that might happen?).

I think this needs to read it out based on the button state. Perhaps could be as simply as checking the props.checked.

Maybe this should be `const updateAnnouncerString = useCallback((cameraOn: boolean) => {`

Then in `onToggleClick`:
```ts
await onToggleCamera(...);
// Camera state is now the opposite of the checked state as we have just successfully toggled the camera
const cameraNowOn = !props.checked;
updateAnnouncerString(cameraNowOn);
```